app/src/main/java/de/xikolo/models/certificates/Certificates.java

```diff
@@ -0,0 +1,17 @@
+package de.xikolo.models.certificates;
+
+import com.squareup.moshi.Json;
+
+import io.realm.RealmObject;
+
+public class Certificates extends RealmObject {
+
+    @Json(name = "confirmation_of_participation")
+    public ConfirmationOfParticipation confirmationOfParticipation;
+
+    @Json(name = "record_of_achievement")
+    public RecordOfAchievement recordOfAchievement;
+
+    @Json(name = "qualified_certificate")
+    public QualifiedCertificate qualifiedCertificate;
+}
```

Comment:
`ConfirmationOfParticipation`, `RecordOfAchievement` and `QualifiedCertificate` could be one model class. No need for three different.

Reply:
Also, just place this model inside the course model class, so you can have a `Course.Certificates` identifier as they are specific to it.